pkgs/checks/lib/src/checks.dart

```diff
@@ -619,7 +621,7 @@ class Rejection {
   /// message. The message will be indented to the level of the expectation in
   /// the description, and printed following the descriptions of any
   /// expectations that have already passed.
-  final String actual;
+  final Iterable<String> actual;
```

Comment:
The doc comment should probably be updated here to describe how multiples are handled

Reply:
Done.